client/devdocs/design/index.jsx

```diff
@@ -50,14 +53,22 @@ import ExternalLink from 'components/external-link/docs/example';
 import FeatureGate from 'components/feature-example/docs/example';
 import FilePickers from 'components/file-picker/docs/example';
 import Collection from 'devdocs/design/search-collection';
+import fetchUsageStats from 'state/usage-stats/actions';
 
-export default React.createClass( {
+let DesignAssets = React.createClass( {
 	displayName: 'DesignAssets',
 
 	getInitialState() {
 		return { filter: '' };
 	},
 
+	componentWillMount() {
+		if ( config.isEnabled( 'devdocs/usage-stats' ) ) {
+			const { dispatchFetchUsageStats } = this.props;
+			dispatchFetchUsageStats();
```

Comment:
One could argue that search is dynamic and the stats are "static" so can be fetched once in the controller and be part of the initial state / props